src/utils/s3index.py

```diff
@@ -99,6 +99,9 @@ def main() -> None:
     args = parser.parse_args()
     types = list(set(args.types))
 
+    if not args.types:
+        types = ["gram3", "text4", "wide8", "hash4"]
```

Comment:
```suggestion
        raise RuntimeError("You must pick at least one index type (see --type)")
```

Reply:
Is no index type also a valid index type? At least it was possible to create a DB like that...
Basically it would be a "Just scan all Files with YARA" index... i dont see a exact use case right now but i could see someone having one.

I would just like to make sure its behaviour  is in sync with the other "index.py"

Anyways, i'd be fine with an error message.

Reply:
We would need to change the function docs which explains the default and mark it as required i guess

Reply:
`Is no index type also a valid index type?` It is, but it's a bit pointless (as you've noticed). It will keep a list of files and you can later add indexes to an existing dataset (`reindex ... with ...` if I remember correctly), but that's a very weird use case so I don't think we have to support it (99% users don't want that I think).

I'll update the docs and ensure other indexes are in sync after merging.